src/js/angular/export/controllers.js

```diff
@@ -207,6 +217,29 @@ exportCtrl.controller('ExportCtrl',
                 }
             };
 
+          /**
+           * Calls the GET graphs endpoint without params, in order to download all the graphs from the BE.
+           */
+          $scope.downloadAllGraphs = function () {
+              if ($scope.getActiveRepository()) {
+                  RDF4JRepositoriesRestService.getGraphs($scope.getActiveRepository())
```

Comment:
There is a RDF4JRepositoriesService, why not using it but instead reaching the rest client directly?

Reply:
The RDF4JRepositoriesService.getGraphs method returns GraphListOptions. I need to extract the filename and content type from the response. However, to address the next comments, I will create a new method for downloading, similar to how it is implemented in ExportRestService.getExportedStatementsAsJSONLD.

Reply:
Refactored